Check config in Logger Adapter

Taffarel opened the pull request:
- Its was implemented a way to check if the needed variables was setted
  in the moment when LoggerAdapter was instanced and if not
  throws an exception

Taffarel committed:
Check config in Logger Adapter

 - Its was implemented a way to check if the needed variables was setted
   in the moment when LoggerAdapter was instanced and if not
   throws an exception

fdiasr commented on src/Logger/QueueLogger.php:
```diff
@@ -93,4 +86,47 @@ public function consume($queue, $consumeTag, $callback)
+            'path' => '/var/log/message-queue-php.log'
+        ];
+
+        if (isset($config['logger']['host'])) {
```
We need throws exceptions if a mandatory parameter is necessary. I dont recommend use default params to connect to a (data) server

Taffarel committed:
Check if params config exists

 - Was removed the default parameters array and instead was
   created an array to verify all required params and test
   against the actual config. If one parameters was not found
   then throw a generic exception;

Commit:
Exception Message improvement
